src/gui/folderman.cpp

```diff
@@ -851,7 +851,6 @@ void FolderMan::slotEtagPollTimerTimeout()
     std::copy_if(folderMapValues.begin(), folderMapValues.end(), std::back_inserter(foldersToRun), [this](Folder *folder) -> bool {
         const auto account = folder->accountState()->account();
         const auto capabilities = account->capabilities();
```

Comment:
should not this one also removed ?

Reply:
Yes, I think so. I just got a warning for the push notifications variable